tests/functional/conftest.py

```diff
@@ -51,6 +51,17 @@ def pytest_configure(config):
             "tests/fixtures/user_permissions.json", user_api_key_name
         )
 
+        # Add organization
+        org_name = "local-custom-model-tests-org"
+        try:
+            # ensure it exists
+            DataRobotUserDatabase.add_organization(environment=env, organization_name=org_name)
+        except ValueError:
+            # already exists
+            pass
```

Comment:
Hey @rvorobii I saw this pattern used elsewhere too, but I think that catching the ValueError here is dangerous and unnecessary. `add_organization()` should return `true` even if the organization already exists, but will raise a `ValueError` if there is no admin API key. This was a change I made before the 3.0.0 release. datarobot/dr-usertool@c6c4143#diff-fe01c37518f26141ae7e0ca068f497f8e44cc2f1e1071195533c2325412c0be5R588-R592